cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapsBundleBuilder.java

```diff
@@ -262,6 +281,14 @@ private DetectedIssue getDetectedIssue(
             groupIdExt.setValue(new StringType(measureReportGroupId));
             detectedIssue.setExtension(Collections.singletonList(groupIdExt));
         }
+        if (!isDocumentMode) {
```

Comment:
Could we conceivably have a third mode in the future, in addition to non document and document modes?  If so, this may be best captured by an enum, not a boolean.